proxy/envoy/testdata/envoy-timeout-rule.json.golden

```diff
@@ -9,62 +9,10 @@
           "config": {
             "codec_type": "auto",
             "stat_prefix": "http",
-            "route_config": {
-              "virtual_hosts": [
-                {
-                  "name": "hello.default.svc.cluster.local:http",
-                  "domains": [
-                    "hello:80",
-                    "hello",
-                    "hello.default:80",
-                    "hello.default",
-                    "hello.default.svc:80",
-                    "hello.default.svc",
-                    "hello.default.svc.cluster:80",
-                    "hello.default.svc.cluster",
-                    "hello.default.svc.cluster.local:80",
-                    "hello.default.svc.cluster.local",
-                    "10.1.0.0:80",
-                    "10.1.0.0",
-                    "10.1.1.0:80",
-                    "10.1.1.0"
-                  ],
-                  "routes": [
-                    {
-                      "prefix": "/",
-                      "cluster": "inbound:80"
-                    }
-                  ]
-                },
-                {
-                  "name": "world.default.svc.cluster.local:http",
-                  "domains": [
-                    "world:80",
-                    "world",
-                    "world.default:80",
-                    "world.default",
-                    "world.default.svc:80",
-                    "world.default.svc",
-                    "world.default.svc.cluster:80",
-                    "world.default.svc.cluster",
-                    "world.default.svc.cluster.local:80",
-                    "world.default.svc.cluster.local",
-                    "10.2.0.0:80",
-                    "10.2.0.0"
-                  ],
-                  "routes": [
-                    {
-                      "prefix": "/",
-                      "cluster": "outbound:world.default.svc.cluster.local:http",
-                      "timeout_ms": 30000,
-                      "retry_policy": {
-                        "retry_on": "5xx,connect-failure,refused-stream",
```

Comment:
This is now output of RDS. It turns out envoy config is identical for all three cases here, and we should be using RDS/CDS output instead.